client/src/test/groovy/jp/co/soramitsu/iroha/java/CommandsTest.groovy

```diff
@@ -54,30 +56,145 @@ class CommandsTest extends Specification {
                 .build()
     }
 
+    /**
+     * Binary code of contract
+     */
+    static def evm_binary_code = "606060405260a18060106000396000f360606040526000357c01000000000000000" +
```

Comment:
Jesus.. 😨 

Could you provide some contract description or the source code in comments as well please

Reply:
Some kind of a getter, I get it from Iroha tests.